bin/compile

```diff
@@ -12,7 +12,7 @@ VENDOR_DIR="$BUILD_DIR/vendor"
 mkdir -p $VENDOR_DIR
 INSTALL_DIR="$VENDOR_DIR/imagemagick"
 mkdir -p $INSTALL_DIR
-IMAGE_MAGICK_VERSION="${IMAGE_MAGICK_VERSION:-6.9.5-7}"
+IMAGE_MAGICK_VERSION="${IMAGE_MAGICK_VERSION:-6.9.5-8.7}"
```

Comment:
Looking at the filename in http://www.imagemagick.org/download/, I think you just want `6.9.5-8` (e.g. without the `.7`)